src/main/kotlin/uk/bfi/uvaudit/event/AuditEventController.kt

```diff
@@ -26,9 +28,16 @@ class AuditEventController(
         logger.error("Audit event structure was invalid", ex)
     }
 
+    @ExceptionHandler(AuthenticationException::class)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
I would suggest this is changed to a `403 Forbidden` instead. The `401 Unauthorized` is used to indicate that the user is not authenticated. In this case, the user is authenticated, but we're denying them access all the same.

https://auth0.com/blog/forbidden-unauthorized-http-status-codes/

Reply:
Done